src/OrchardCore/OrchardCore.ResourceManagement.Core/OrchardCore.ResourceManagement.Core.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+
+  <PropertyGroup>
+    <RootNamespace>OrchardCore.ResourceManagement</RootNamespace>
+    <!-- NuGet properties-->
+    <Title>OrchardCore Roles Core</Title>
+    <Description>$(OCCMSDescription)
```

Comment:
use OC framework's desc

Reply:
Even when this is a core project not an abstraction?

Reply:
Yes it’s not related to cms

Reply:
Done!